code-challenge/Models/Discover/Discover.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+struct Discover: Decodable {
+    let identifier: Int
+    let title: String
+    let votes: Int
+    let isVideo: Bool
+    let avarageVote: Double
+    let popularity: Double
+    let poster: String
+    let originalLanguage: String
+    let originalTitle: String
+    let genre: [Int]
```

Comment:
maybe `genres` (plural) would be a better name here given this is an array

Reply:
you are right 👍